code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -777,7 +786,14 @@ var/global/list/robot_modules = list(
 					LANGUAGE_SIIK_MAAS = 1,
 					LANGUAGE_SKRELLIAN = 1,
 					LANGUAGE_GUTTER = 1,
-					LANGUAGE_ROOTSONG = 1
+					LANGUAGE_VAURCESE = 0,
+					LANGUAGE_ROOTSONG = 1,

```

Comment:
Same as the above
```suggestion
					LANGUAGE_ROOTSONG = 0,
```

Reply:
They were able to speak it before, no reason to change it?